tests/performance/requirements.txt

```diff
@@ -0,0 +1,11 @@
+gevent==20.5.2
+junitparser==1.4.1
+git+https://github.com/maheshambule/vjunit.git#egg=vjunit
```

Comment:
Just curious, what are we using this for?

Reply:
We are using vjunit for generating HTML report from Junit XML. There was a bug in it so I had fixed in my fork and using it here.